packages/devextreme/js/localization/messages/ar.json

```diff
@@ -48,6 +48,9 @@
         "dxList-listAriaLabel-deletable": "Deletable items",
         "dxListEditDecorator-delete": "حذف",
         "dxListEditDecorator-more": "المزيد",
+        "dxList-halfChecked": "half checked",
```

Comment:
List itself cannot be checked, only its selectAll checkbox. I think these variables should be like this:
```json
 "dxList-selectAll-indeterminate": "half checked",
 "dxList-selectAll-checked": "checked",
 "dxList-selectAll-notChecked": "not checked",
```
But before applying changes let's get approve by @vladaskorohodova

Reply:
Please redo as follows:
```
 "dxList-selectAll-indeterminate": "Half-checked",
 "dxList-selectAll-checked": "Checked",
 "dxList-selectAll-notChecked": "Not checked",
```

Reply:
[54d0fca](https://github.com/DevExpress/DevExtreme/pull/28070/commits/54d0fca21aee869cdc5fd4ccaf7d633ab9fc0873)